eoscostarica/eos-rate
# feat(ci): configs for staging env

### GH Issue

#107

### Steps to test

1. you can cd `services/frontend` and `yarn start:staging`
1. netlify build for this PR should connect to `http://190.10.8.205:8088/v1alpha1/graphql`

### Checklist
